docs/resources/Template.md

```diff
@@ -0,0 +1,114 @@
+# Template Resource
+
+### Template Object
+
+Represents a code that when used, creates a new guild duplicated from an existing one.
```

Comment:
```suggestion
Represents a code that when used, creates a new guild based on the template data.
```

Reply:
IMO that wording seems to imply that the template is an individual thing with its own separate data, while it's actually just a server's data.

Reply:
A server template can go out of sync with a server's data. Try creating a template in a server than adding or removing a channel. This means that they do include their own data.